src/components/Layout/HomeContent.js

```diff
@@ -1537,7 +1529,7 @@ function Cover({background, children}) {
         src={background}
         width={500}
         height={263}
-        alt=""
+        alt="cover"
```

Comment:
Is this useful to add? Guess I'm not sure the context of when this is displayed

Reply:
I pushed a commit to remove it. thanks